### DISCUSSION TITLE
Is it possible to set default title for links

### DISCUSSION THREAD

**Discussion by sytolk** [Milkdown/milkdown#1327] [Q&A]
If I have markdown link like `[Awesome website](https://example.com "Visit this awesome website!")` title "Visit this awesome website!" appears in active and preview mode like a tooltip. If there is no title in milkdown can we have option to show tooltip with href instead? I was remember that in milkdown v6 works without titles but show in tooltip href. Now milkdown v7 support title attr but option to show href like a title miss me.

**Comment by Saul-Mirone** [comment]
It's possible, have you check [this](https://milkdown-storybook.vercel.app/?path=/story/crepe-main--width-default-value)? Maybe you want to use the link tooltip component.

**Comment by sytolk** [comment]
Link tooltip is for edit mode. I can't see tooltip in preview mode (read only switch) onMouseOver ? There is existing tooltip if link have title props. Is it possible to extend this default title tooltip to show href if title is missing ?

**Reply by Saul-Mirone** [reply]
It is still possible I believe. But you will need to implement the UI part. The update commands can work out of box.
